CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [11.4.0](https://github.com/theforeman/puppet-candlepin/tree/11.4.0) (2021-05-04)
+
+[Full Changelog](https://github.com/theforeman/puppet-candlepin/compare/11.3.0...11.4.0)
+
+**Implemented enhancements:**
+
+- Allow Puppet 7 compatible versions of mods [\#193](https://github.com/theforeman/puppet-candlepin/pull/193) ([ekohl](https://github.com/ekohl))
```

Comment:
It should be noted that those versions technically drop Puppet 5 support. Puppetlabs has this odd policy where they sort of support it for 6 months but don't list it in metadata.json. That's why I pushed these all after we had releases for branching. That's why I wasn't that trigger happy to releases it just yet.